docs/topics_5/rn-new-features.adoc

```diff
@@ -5,13 +5,60 @@
 
 This section describes the new features of the {ProductName} {DocInfoProductNumber}.
 
-Updated {WebName} interface::
+== Updated {WebName} interface
 The {WebName} has been redesigned with PatternFly 4. In addition to its general improved appearance, the {WebName} adds elements for project creation and management and updated transformation target icons.
 
-{ProductName} Operator for OpenShift Container Platform::
+== {ProductName} Operator for OpenShift Container Platform
 The current release includes the {ProductName} Community Operator, which installs the {WebName} on OpenShift Container Platform 4.5 and later.
 
-Greater support for migrating from Spring Boot to Quarkus::
-The current release provides greater support for migrating from Spring Boot to Quarkus with the addition of 22 rules.
-+
+== Greater support for migrating from Spring Boot to Quarkus
+The release contains numerous rules to support migrating from Spring Boot to Quarkus.
+
 The new rules focus on the Spring Boot APIs that are implemented in Quarkus Spring extensions, where Spring Boot dependencies can be replaced by equivalent Quarkus extensions. Spring artifacts that do not have Quarkus equivalents are also reported.
+
+Numerous other technologies, for which there are equivalent Quarkus extensions, are also identified by the new rules. These include:
```

Comment:
Remove passive voice. 
Not sure "also" adds anything.
If this is a complete list, you should not have "These include:"